index.html

```diff
@@ -25,10 +25,6 @@
     <meta name="twitter:description" content="Rapido: semantic HTML elements for your writings.">
     <!-- <meta name="twitter:image" content="https://nextbitlabs.github.io/Rapido/images/intro.png"> -->
 
-    <meta itemprop="name" content="Rapido">
-    <meta itemprop="description" content="Rapido: semantic HTML elements for your writings.">
```

Comment:
Why do you have to remove this meta tags?

Reply:
Good question! The HTML validator emits an error on those lines: **Validation error: “The itemprop attribute was specified, but the element is not a property of any item**. Actually, `itemprop` attributes [refer](https://schema.org/docs/gs.html#microdata_itemscope_itemtype) to `itemscope` and `itemtype` attributes, which are not present.

Therefore there is the option to include valid tags, but adding microdata is the scope of issue #84, so I prefer to address this problem in that issue.